app/views/topics/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Topics#create</h1>
+<p>Find me in app/views/topics/create.html.erb</p>
```

Comment:
I am not sure as I don't really know how Topics is going to flesh out.  I kept the Topic  create, destroy, edit and update views as I am not sure whether things will change that will require needing them.  For example--in the end are only Admins going to be allowed to create topics, edit or destroy them?  If so, then perhaps separate views for these are needed.  Maybe the philosophy is to not retain stuff in case you need it, but create it at the time you do.  Let's talk about that.  I think some of this can be answered by which direction the app is going?  Under what circumstances are topics created?  Who can delete a topic? Any user?  Or the user who created it.  I understand that Topics will include Bookmarks from multiple users and so can a user who created a Topic delete it even if other uses have Bookmarks under it?